modules/ROOT/pages/6.0-release-notes-core-changes.adoc

```diff
@@ -0,0 +1,296 @@
+* values changed in a default {productname} 6.0 instance.
+* previously deprecated and now removed items.
+* never documented, and consequently never supported, items that have now been removed.
+
+
+* xref:core-changes[Core changes]
+* xref:things-we-renamed[Things we renamed]
+* xref:default-value-changes[Default value changes]
+* xref:previously-deprecated-items-now-removed[Previously deprecated items now removed]
+* xref:previously-undocumented-items-removed-without-prior-deprecation[Previously undocumented items removed without prior deprecation]
+
+
+// tag::core-changes[]
+[[core-changes]]
+== Core changes
+
+. {productname} 6.0 now requires a browser capable of running https://tc39.es/ecma262/2020/[ECMAScript 2020] (ES2020).
++
+NOTE: In particular, this means {productname} 6.0 does not support Internet Explorer 11.
++
+. No Premium Plugins being released with {productname} 6.0 are backwards compatible with earlier versions of {productname}.
+. {productname} 6.0 is now available in three further language:
+* Hindi (hi);
+* Malay (ms); and
+* Vietnamese (vi)
+
+// end::core-changes[]
+
+
+// tag::things-we-renamed[]
+[[things-we-renamed]]
+== Things we renamed
+
+[cols="1,1,1,1,1"]
+|===
+| old name                            | new name                         | element                             | notes                                                                                                                                                                                                                                                                                                         | for more information
+
+| `blockformats`                      | `blocks`                         | Menu item                           | No behavioural changes.                                                                                                                                                                                                                                                                                       |
+
+| `formatselect`                      | `blocks`                         | Toolbar item                        | No behavioural changes.                                                                                                                                                                                                                                                                                       |
+
+| `imagetools`                        | `editimage`                      | Plugin                              | NB: _Image Tools_ is now _Edit Image_, a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].                                                                                                                                                                                                   | see xref:new-and-improved-plugins-imagetools[EditImage]
```

Comment:
I mentioned this elsewhere, but just to make sure it's not missed the plugin name should be `Enhanced Image Editing` (it was changed unexpectedly late last week).

Reply:
Also we should be using the attributes defined globally for these URLs so that it can easily be changed if the URL changes later. e.g.:
```suggestion
| `imagetools`                        | `editimage`                      | Plugin                              | NB: _Image Tools_ is now _Enhanced Image Editing_, a link:{plugindirectory}#productivity[Premium plugin].                                                                                                                                                                                                   | see xref:new-and-improved-plugins-imagetools[Enhanced Image Editing]
```

Reply:
Did also wonder if "see" should be capitalised?